### PR TITLE
Print on death via fall damage.

### PR DESCRIPTION
**Pull request by Hexwolf**
Prints how to combat roll if you die from fall damage.

### CODE REVIEW

**Review by justinmuskopf — DISMISSED** [review]
LGTM

**Review by sarahsturgeon — APPROVED** [review]
As long as it's been tested

**Comment by Hexwolf** [comment]
> As long as it's been tested

Chat print feature is currently untested.

**Comment by sarahsturgeon** [comment]
retest this please

File: lua/autorun/server/sv_custom_fall_damage.lua [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+hook.Add( "GetFallDamage", "CFC_FallDamage", getFallDamage )
 
--- Bastards arbitrarily returning "10" before we can get to it
-hook.Remove( "GetFallDamage", "prone.FallDamage" )
-hook.Remove( "GetFallDamage", "Get Fall Damage" )
 
+local function onPlayerDeath( ply, attacker, dmg )
+    if not dmg:IsFallDamage() then return end
+    if not IsValid( ply ) and ply:IsPlayer() then return end
+
```
Comment (sarahsturgeon) [inline lua/autorun/server/sv_custom_fall_damage.lua]:
```suggestion
```

File: lua/autorun/server/sv_custom_fall_damage.lua [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
+local SPEED_MULTIPLIER = 0.175
+local SPEED_OFFSET = 141.75
+
+local DEATH_MESSAGE = "Did you know you can reduce the amount of damage you take from falling by crouching and looking down before you impact a surface?"
 
-local function calculateDamage( ply, speed )
+local function getFallDamage( ply, speed )
+    -- Based on CS:S fall damage algorithm
     local damage = ( SPEED_MULTIPLIER * speed ) - SPEED_OFFSET
     damage = math.max( 0, math.ceil( damage ) )
```
Comment (sarahsturgeon) [inline lua/autorun/server/sv_custom_fall_damage.lua]:
```suggestion
```

**Review by sarahsturgeon — DISMISSED** [review]

**Review by sarahsturgeon — APPROVED** [review]

**Pull request merged**